src/features/orders/reducer/filters.ts

```diff
@@ -0,0 +1,50 @@
+import { createReducer } from '../../../share/dateRangePicker'
```

Comment:
do you need these reducers/selectors? where do you use them?

Reply:
Yes, it's unnecessary logic. I fotgot to delete